internal/formats/spdx22json/model/annotation.go

```diff
@@ -12,7 +10,7 @@ const (
 type Annotation struct {
 	// Identify when the comment was made. This is to be specified according to the combined date and time in the
 	// UTC format, as specified in the ISO 8601 standard.
-	AnnotationDate time.Time `json:"annotationDate"`
```

Comment:
honest question: is this a fix? or related to the digests work?

Reply:
This was not related. This snuck in because I had initially included digests as an annotation. I can revert it, but it's worth mentioning that `time.Time` does not by default print the ISO 8601 standard. To get that standard we would call something like `time.Now().Format(time.RFC3339)` which would return a string.